regparser/layer/terms.py

```diff
@@ -3,7 +3,12 @@
 from itertools import chain
 import re
 
-from inflection import pluralize
+import inflection 
+try:
+    del inflection.PLURALS[inflection.PLURALS.index(('(?i)(p)erson$', '\\1eople'))]
+except ValueError:
+    pass
```

Comment:
Yeah, I think it's future work. For the defined term "person", I think this is ok for now. 

I thought about submitting a PR to inflection to return a list of possible plurals instead of returning the first one it finds, but that'd probably break anything and everything else that uses the library.